hack/verify-style.sh

```diff
@@ -23,8 +23,8 @@ trap 'rc=$?' ERR
 # Go to the root of the repo
 cd "$(git rev-parse --show-cdup)"
 
-GOFILES=$(find . -path ./vendor -prune -o -name '*.go' | grep -v vendor | grep -v pkg/operator/assets)
-GOPKGS=$(go list ./... | grep -v '/vendor/' | grep -v '/generated/' | grep -v pkg/operator/assets)
+GOFILES=$(find . -path ./vendor -prune -o -name '*.go' | grep -v vendor)
+GOPKGS=$(go list ./cmd... ./pkg/...)
```

Comment:
the old version was written before `go` knew about `vendor`. Just

    GOPKGS=$(go list ./...)

will work too. (And picks up the `version` subdir, which I'm not sure if you had meant to omit)

Reply:
We need to skip `dependencymagnet`, hence the changes here. (And version will go away soonish)